### DISCUSSION TITLE
학습이 안되는데 뭘 잘못 했을까요?

### DISCUSSION THREAD

**Discussion by ppskj178** [boostcampaitech2/object-detection-level2-cv-06#62] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20790778/135897358-12604d50-9172-4025-ad35-4f396319b5b5.png)

코드는 valid_search_faster-rcnn 브랜치에 푸쉬해놨는데
고쳐지질 않네요...

**Comment by jaehwan-AI** [comment]
저도 동일 문제를 해결 중에 있습니다.. 문제 해결 시 공유하겠습니다

한 가지 의심이 되는 점은 Adam으로 optimizer를 변경하니까 학습이 잘 안되는 것 같은데 다시 SGD 세팅으로 시도해보는 건 어떨까 합니다.

**Comment by ppskj178** [comment, marked as answer]
회의로 해결됨 lr을 매우 낮게 주는걸로...
